core/templates/dev/head/pages/skill_editor/SkillEditorNavbarDirective.js

```diff
@@ -49,6 +47,32 @@ oppia.directive('skillEditorNavbar', [
             return UndoRedoService.getChangeCount();
           };
 
+          $scope.selectQuestionsTab = function() {
+            // This check is needed because if a skill has unsaved changes to
```

Comment:
Should we update the e2e tests to include a scenario like this?

Reply:
I am not sure the new structures editors have e2e tests yet. I don't think they do.

Reply:
OK -- can you file a bug to track please?

Reply:
Created [issue](https://github.com/oppia/oppia/issues/6085) for creating e2e tests.